### DISCUSSION TITLE
picture	string

### DISCUSSION THREAD

**Discussion by vgr28** [idaho/hassio-trash-card#482] [General]
Hi, gibt es Anforderung an das hinterlegte Bild wie Abmessung, Dateiformat, Dateigrösse? Bei mir werden keine Bilder anzeigt. Mit Browser http://xxx.xxx.x.xxx:8123/local/abfall/green.png funktioniert es

**Comment by anefeldk** [comment]
Ich habe in Paint ein 222x222 Pixel Blatt erstellt und da mein Icon eingefügt. 222x222 aus dem Grund da mein eigentliches Icon knall die Höhe von 222 Pixel hatte. Da die Card das Bild zu einem Quadrat verzerrt habe ich es auf 222x222 angepasst. So wird nichts verzerrt. 
Gespeichert habe ich die png Datei im Ordner /homeassistant/www/abfall
Die Datei hat eine Größe von ca. 36 KB

**Reply by vgr28** [reply]
Könntest du mal ein Bild hier einstellen, dann könnte ich es mal probieren ob es bei mir an der Größe liegt. Habe eigentlich den gleichen Pfad wie du aber es funktioniert nicht.

**Comment by anefeldk** [comment]
![green](https://github.com/user-attachments/assets/daf52019-51d3-4866-b785-7f0fab0dee78)

**Reply by vgr28** [reply]
danke, funktioniert bei mir leider auch nicht. Scheint eine andere Ursache zu haben..

**Comment by idaho** [comment]
## German - English below

Eigentlich gibt es da keine Anforderungen. Am besten quadratisch. Also Höhe und Breite gleich und natürlich solle es sich um eine Bild Datei handeln (jpg, jpeg, png, etc)
Schau mal ob du es im richtigen Ordner abgelegt hast.

`/www/[bildname].[erweiterung]`

also z.B.

`/www/grune-tonne.jpg`

Dann einfach nur noch als Bild URL

`/local/grune-tonne.jpg`

verwenden.

Könntet ihr demnächst auf englisch schreiben? Dann können andere die vllt kein deutsch können auch helfen oder nach sehen wenn sie das gleiche Problem haben. Danke

## English 

There are actually no requirements. Square is best. So height and width should be the same and of course it should be an image file (jpg, jpeg, png, etc)
Check whether you have placed it in the correct folder.

`/www/[image name].[extension]`

e.g.

`/www/grune-tonne.jpg`

Then simply use the image URL

`/local/grune-tonne.jpg`

as the image URL.

**Reply by vgr28** [reply]
I use exact your example but it did not work. my HA is running on Proxmox integration. Could that be the reason why I don't see the pictures?

**Reply by idaho** [reply]
Could you share your configuration?

**Reply by vgr28** [reply]
event_grouping: true
drop_todayevents_from: "10:00:00"
next_days: 23
pattern:
  - icon: mdi:flower
    color: lime
    type: organic
    picture: /local/abfall/green2.png
  - icon: mdi:newspaper
    color: blue
    type: paper
    picture: /local/abfall/blue.png
  - icon: mdi:recycle-variant
    color: amber
    type: recycle
  - icon: ""
    color: ""
    type: waste
    picture: /local/abfall/black.png
    pattern_exact: false
  - icon: mdi:dump-truck
    color: yellow
    type: others
day_style: default
day_style_format: yyyy.MM.dd
card_style: card
alignment_style: left
color_mode: background
items_per_row: 1
refresh_rate: 60
with_label: true
layout: default
type: custom:trash-card
entities:
  - calendar.ics
filter_events: false
